auth/config/config.go

```diff
@@ -111,14 +111,6 @@ var (
 						Scopes:        []string{"all", "offline", "access_token"},
 						Public:        true,
 					},
-					"flytepropeller": {
-						ID:            "flytepropeller",
-						Secret:        []byte(`$2a$06$pxs1AkG81Kvrhpml1QiLSOQaTk9eePrU/7Yab9y07h3x0TglbaoT6`), // = "foobar"
```

Comment:
Should we leave the settings for flytepropeller and only remove the secret (if that's a valid config)

Reply:
I don't think so?  I feel like I can see a future version where then an empty password is somehow valid.

Reply:
Maybe generate a random long string and hash it and put the hash there and never store the string anywhere? I just don't want the default to be "wrong"... maybe just inaccessible...